src/types/events/message-events.ts

```diff
@@ -111,6 +113,7 @@ export interface MessageRepliedEvent {
 export interface ThreadBroadcastMessageEvent {
   type: 'message';
   subtype: undefined;
+  channel: string;
   message: {
```

Comment:
this field does not exist. the fields exist at top-level. can you verify the actual payload?

Reply:
It turns out the actual payload looks quite different than what's here. I'll update with what I've got, but a follow-up on which are optional would be helpful, if you've got insight on that.